test/new-e2e/tests/agent-subcommands/diagnose/diagnose_common_test.go

```diff
@@ -46,6 +46,7 @@ var commonSuites = []string{
 	"connectivity-datadog-autodiscovery",
 	"connectivity-datadog-core-endpoints",
 	"connectivity-datadog-event-platform",
+	"port-conflict",
```

Comment:
Does it need to be removed from the linux specific files ?

Reply:
I checked and there are a few lines adding `port-conflict` to the list in `test/new-e2e/tests/agent-subcommands/diagnose/diagnose_nix_test.go`